scripts/controls.js

```diff
@@ -1,19 +1,14 @@
-function keyEvents() {
-	if (keyDown(controls.play)) {
+function checkKey(key) {
+	// checks an incoming key
+	if (key == keys[controls.play]) {
```

Comment:
I think keyDown(controls.play) looks a little cleaner than this, but that's alright

Reply:
Ah sorry, you definitely have a point there! It's something I tried out at the start of my fiddling and completely forgot about. Feel free to add it back in if you want!